src/future.ts

```diff
@@ -192,16 +193,17 @@ export function fromPromise<A>(p: Promise<A>): Future<A> {
  * @private
  */
 export class BehaviorFuture<A> extends SinkFuture<A> implements Observer<A> {
+  node= new Node(this);
```

Comment:
Missing space before `=`. Prettier should fix that 😄

Reply:
I'll run prettier on everything